### PR TITLE
[TACHYON-698] Remove getToMasterHeartBeatIntervalMs

### PR DESCRIPTION
**Pull request by uronce-cc**
https://tachyon.atlassian.net/browse/TACHYON-698

### CODE REVIEW

**Comment by calvinjia** [comment]
@uronce-cc Thanks for the patch, if the heartbeat period is read multiple times, could you store it in a variable?

**Comment by uronce-cc** [comment]
@calvinjia put into a lazy initialized variable if needed. Commits are rebased.

File: integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java [hunk 1]
```diff
@@ -72,7 +81,7 @@ public void pinFileTest() throws Exception {
 
     // Pin the file
     mTFS.pinFile(fileId);
-    CommonUtils.sleepMs(LOG, CommonUtils.getToMasterHeartBeatIntervalMs(mWorkerConf) * 3);
+    CommonUtils.sleepMs(LOG, heartbeatInterval() * 3);
```
Comment (apc999) [inline integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java]: why not setting `mWorkerToMasterHeartbeatIntervalMs` in `before`? It is really unusual to set a member variable in its first appearance in a test case and reference this member later on in the same test case---member variable in this case ca be replaced by a local variable.

Comment (uronce-cc) [inline integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java]: Moved to `before`.

**Comment by apc999** [comment]
+1 LGTM

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**